debugtools/DDR_Artifacts/specs/29/aix_ppc

```diff
@@ -7624,6 +7624,8 @@ C|J9VMCONSTANTPOOL_JAVALANGVIRTUALMACHINEERROR
 C|J9VMCONSTANTPOOL_JAVALANGVOID
 C|J9VMCONSTANTPOOL_JAVAUTILCONCURRENTLOCKSABSTRACTOWNABLESYNCHRONIZER
 C|J9VMCONSTANTPOOL_JAVAUTILCONCURRENTLOCKSABSTRACTOWNABLESYNCHRONIZER_EXCLUSIVEOWNERTHREAD
+C|J9VMCONSTANTPOOL_JAVANIOBITS_COUNT
```

Comment:
You shouldn't need to modify any of the DDR specs or the superset.

Reply:
Done. It was faster to make a new PR.

New PR: https://github.com/eclipse/openj9/pull/1508